draft-ietf-rats-architecture.md

```diff
@@ -607,12 +607,13 @@ Produced By: Reference Value Provider.
 
 # Topological Models {#overview}
 
-{{dataflow}} shows a basic model for communication between an Attester,
+{{dataflow}} shows an abstract model for communication between an Attester,
```

Comment:
Is it necessary to change "basic" to "abstract"?
"An abstract model" makes me feel that this model is not practical. But actually, it is a viable model, so I think it's a basic model.